examples/compose/column_selector_callables.py

```diff
@@ -0,0 +1,69 @@
+"""
+=======================
+Select Column Functions
```

Comment:
I think the existing examples would benefit from this

Reply:
I will drop this new example then, and enhance the already existing one for `ColumnTransformer`.